### ISSUE TITLE
Issue with Duckling

### ISSUE THREAD

**Issue by znat** [RasaHQ/rasa_nlu#512]
**rasa NLU version** (e.g. `0.7.3`): 0.10

Proabably not a rasa issue. But it affects it. I only have _time_ as `duckling_dimensions`

The problematic sentence is : "Besoin d'une réparation de carrosserie à pound cove"

Where it breaks: `wrapper.py` l.58

```python
    def _parse(self, input_str, dim=None, reference_time=''):
        result = []
        duckling_result = self.duckling.parse(
            input_str, self.language, dim_filter=dim,
            reference_time=reference_time)
        for entry in duckling_result:
            result_entry = self._dims[entry[u'dim']](entry)  # here: KeyError: u'leven-unit'
            result.append(result_entry)
        return result
```

where `entry` is: `{u'body': u'pound', u'dim': u'leven-unit', u'end': 44, u'value': {u'value': u'pound'}, u'start': 39}`

Duckling seems to be loaded with all dimension. Not sure if that's an error, but `leven-unit` is not here.

![screen shot 2017-08-02 at 3 43 38 pm](https://user-images.githubusercontent.com/3694634/28891697-6a7d4a06-7799-11e7-80a5-2f311db76d27.png)

**Comment by tmbo** [comment]
@znat which version of the python duckling wrapper are you using?

**Comment by znat** [comment]
I tried with the last two. 1.7.1 and 1.7.2

**Comment by tmbo** [comment]
I created a PR to fix this in the duckling wrapper: https://github.com/FraBle/python-duckling/pull/24

**Comment by FraBle** [comment]
The PR has been merged and released to PyPi as [v1.7.3](https://pypi.python.org/pypi/duckling/1.7.3).

**Comment by tmbo** [comment]
thanks so much @FraBle
